packages/jest/src/jest.ts

```diff
@@ -30,15 +30,28 @@ function getCacheKey(
 
 export interface StylableJestConfig {
     stylable?: Partial<StylableConfig>;
+    configPath?: string;
 }
 
 export const createTransformer = (options?: StylableJestConfig) => {
+    let resolveModule;
+
+    if (options?.configPath && !options.stylable?.resolveModule) {
+        const { defaultConfig } = require(options.configPath);
```

Comment:
would be nice to catch any errors in require or call to user call and through a clear error of where something failed (true for similar places in this PR)

Reply:
Added try/catch and matching test